client/components/download-link/renderers/docx-renderer.js

```diff
@@ -578,6 +584,21 @@ export default (application, sections, values, updateImageDimensions) => {
       return renderAnimalQuantities(doc, values, noSeparator);
     }
 
+    if (field.type === 'duration') {
+      return renderDuration(doc, value, noSeparator);
+    }
+
+    if (field.type === 'holder') {
+      const firstName = application.licenceHolder ? application.licenceHolder.firstName : '';
+      const lastName = application.licenceHolder ? application.licenceHolder.lastName : '';
+      return renderText(doc, `${firstName} ${lastName}`, noSeparator);
+    }
+
+    if (field.type === 'licenceNumber') {
+      const licenceNumber = application.licenceNumber ? application.licenceNumber : '';
+      return renderText(doc, `${licenceNumber} `, noSeparator);
+    }
```

Comment:
Some field types are handled by explicit `if` statements, and others inside a `switch`. Can that be made consistent?

Reply:
I wasn't sure if these were intentionally moved above
```
 if (isUndefined(value) || isNull(value)) {
   return renderNull(doc, noSeparator);
 }
```
to avoid the call to `renderNull`?

Reply:
@wheelsandcogs yes, that is exactly why.  But I agree with @lennym. Replaced ifs with a switch.